cli/src/helpers/logNextSteps.ts

```diff
@@ -28,5 +28,9 @@ export const logNextSteps = ({
     );
   }
 
+  if (packages?.nextAuth.inUse) {
+    logger.warn("  set NEXTAUTH_SECRET in .env");
+  }
```

Comment:
I don't know if we need to add the line about the secret to CLI - we are including a default secret now, and our .env (which users need to check when deploying to Vercel etc because they need to know which env vars to specify there) already includes instructions for how to generate a production ready secure secret.

But I could be convinced otherwise if you or someone else feels strongly about this

Reply:
I think it can stay

Reply:
Since we are already predefining the secret, maybe it can be reworded to "  set your own NEXTAUTH_SECRET in .env"?

Reply:
Or just "change"

Reply:
An alternative approach is to change the schema so it's only required in prod, and have the NEXTAUTH_SECRET commented out in the .env